packages/core/src/Toast/Toast.tsx

```diff
@@ -63,6 +63,15 @@ const LabelContainer = styled.div`
 
   overflow: hidden;
   text-overflow: ellipsis;
+
+  word-wrap: break-word;
+  /* stylelint-disable-next-line property-no-vendor-prefix */
```

Comment:
@lekoaf  
- Removed vendor prefixes. Tested on chrome and Firefox and they are applied by default.
- Removed the hyphen as well because of the poor support